engine/source/materials/fail/johnson_cook/fail_johnson_ib.F

```diff
@@ -53,37 +53,37 @@ SUBROUTINE FAIL_JOHNSON_PI(
       my_real                     ,INTENT(IN)    :: TIME    ! current time
       my_real ,DIMENSION(NUPARAM) ,INTENT(IN)    :: UPARAM  ! failure model parameter array
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: DPLA    ! plastic strain increment
-      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: EPSP    ! strain rate
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: EPSD    ! strain rate
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: TSTAR   ! normalized temperature
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: OFF     ! element desactivation flag
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: SIGNXX  ! stress component
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: SIGNXY  ! stress component
       my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: SIGNZX  ! stress component
       INTEGER ,DIMENSION(NEL)     ,INTENT(INOUT) :: FOFF    ! integration point desactivation flag
       my_real ,DIMENSION(NEL)     ,INTENT(INOUT) :: DFMAX   ! maximum damage
-      my_real ,DIMENSION(NEL)     ,INTENT(OUT)   :: TDEL    ! desactiovation time
+      my_real ,DIMENSION(NEL)     ,INTENT(OUT)   :: TDEL    ! desactivation time
```

Comment:
Same remark, 
TDEL may not be fully initialized after the call of this routine

Reply:
I have changed both commits, it's safe now. thanks